app/src/main/res/values/strings.xml

```diff
@@ -16,7 +16,7 @@
     <string name="main_not_numeric">%1$s is not a number.</string>
     <string name="main_invalide_trainerlvl">Invalid Trainer Level!</string>
     <string name="notification_title">GoIV Running - Level %1$s</string>
-    <string name="notification_text">Tap to open</string>
+    <string name="notification_open_app">Open app</string>
```

Comment:
Is there a behavior change that requires changing (and retranslating) the string?